force-app/volunteers/reports/VolunteerReports/AllVolunteersLastMonth.report-meta.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Report xmlns="http://soap.sforce.com/2006/04/metadata">
+    <columns>
+        <field>FK_NAME</field>
+    </columns>
+    <columns>
+        <field>VolunteerHoursLog__c.StartDateTime__c</field>
+    </columns>
+    <columns>
+        <aggregateTypes>Sum</aggregateTypes>
+        <field>VolunteerHoursLog__c.Hours__c</field>
+    </columns>
+    <description>Volunteers who logged hours last month.</description>
+    <filter>
+        <criteriaItems>
+            <column>VolunteerHoursLog__c.StartDateTime__c</column>
+            <columnToColumn>false</columnToColumn>
+            <isUnlocked>true</isUnlocked>
+            <operator>equals</operator>
+            <value>LAST_MONTH</value>
+        </criteriaItems>
+        <language>en_US</language>
+    </filter>
+    <format>Summary</format>
+    <groupingsDown>
+        <dateGranularity>Day</dateGranularity>
+        <field>VolunteerHoursLog__c.VolunteerContact__c</field>
+        <sortOrder>Asc</sortOrder>
+    </groupingsDown>
+    <name>Volunteers Last Month</name>
```

Comment:
I'd like to see the developer names match the report titles. Would All Volunteers Last Month make a better title?